sympy/physics/vector/frame.py

```diff
@@ -190,9 +190,9 @@ def __init__(self, name, indices=None, latexs=None, variables=None):
             self.latex_vecs = latexs
         self.name = name
         self._var_dict = {}
-        #The _dcm_dict dictionary will only store the dcms of parent-child
-        #relationships. The _dcm_cache dictionary will work as the dcm
-        #cache.
+        #The _dcm_dict dictionary will only store the dcms of immediate parent-child
```

Comment:
Would "adjacent" be better than "immediate"? I don't really know what immediate means in this context.

Reply:
By immediate I mean, chain parents-child relations won't be stored.
```
A.orient(B)
C.orient(A
```
Then `C._dcm_dict` won't have dcm wrt B

Reply:
@moorepants I think "adjacent" would be better.